index.bs

```diff
@@ -38,6 +38,9 @@ spec: ecma262; urlPrefix: https://tc39.github.io/ecma262/
     type: dfn
         text: Promise; url: #sec-promise-objects
         text: AsyncIterator; url: #sec-asynciterator-interface
+spec: fileapi; urlPrefix: https://w3c.github.io/FileAPI/
```

Comment:
since this spec isn't (normatively) depending on this dfn yet, would it make sense to just use a plain <a> hyperlink to the right URL directly in the issue text, rather than using anchors to work around this term not being exported? After all, I assume that if this spec does end up normatively depending on this definition we'd fix FileAPI to export it rather than defining it in anchors here. (I think explicitly specifying what spec a term is defined in at the place you reference it also lets you link to definitions that aren't exported, although not sure what the preferred way is these days to link to non-exported terms).

Reply:
Done. (FWIW, I think of anchor sections as the monkey patch that can get removed when the target spec is updated, but since it's not a normative reference a hyperlink is fine.)